lib/SPIRV/libSPIRV/SPIRVEnum.h

```diff
@@ -53,6 +54,9 @@ typedef uint32_t SPIRVId;
 #define SPIRVID_MAX ~0U
 #define SPIRVID_INVALID ~0U
 #define SPIRVWORD_MAX ~0U
+static constexpr unsigned SpirvWordSize =
+    static_cast<unsigned>(sizeof(SPIRVWord));
+static constexpr unsigned SpirvWordBitWidth = SpirvWordSize * CHAR_BIT;
```

Comment:
Can we explicitly say that word size is 32 bits and add a comment with reference to the spec p 2.2.1
```suggestion
static constexpr unsigned SpirvWordBitWidth = 32;
```

Reply:
Done